elasticsearch/backoff.go

```diff
@@ -0,0 +1,55 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package elasticsearch
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+
+	"github.com/elastic/beats/v7/libbeat/outputs/elasticsearch"
+)
+
+type backoffFunc func(int) time.Duration
+
+var (
+	// DefaultBackoffConfig is the default backoff configuration used for
+	// es clients.
+	DefaultBackoffConfig = elasticsearch.Backoff{
+		Init: time.Second,
+		Max:  time.Minute,
+	}
+
+	// DefaultBackoff is an exponential backoff configured with default
+	// backoff settings.
+	DefaultBackoff = exponentialBackoff(DefaultBackoffConfig)
+)
+
+func exponentialBackoff(b elasticsearch.Backoff) backoffFunc {
+	retryBackoff := backoff.NewExponentialBackOff()
```

Comment:
According to https://github.com/cenkalti/backoff/blob/c2975ffa541a1caeca5f76c396cb8c3e7b3bb5f8/exponential.go#L77-L78, there's a default jitter (RandomizationFactor) and mulitplier is 1.5. Did you change your mind about adding jitter, or should we set that to zero? And should we explicitly set the multiplier to 2 to match libbeat?

Reply:
Let's explicitly set the multiplier, and while we're in there, we can drop the randomization